internal/gc.h

```diff
@@ -141,25 +142,11 @@ struct rb_objspace; /* in vm_core.h */
 # undef RB_NEWOBJ_OF
 #endif
 
-#define RVALUE_SIZE (sizeof(struct RBasic) + sizeof(VALUE[RBIMPL_RVALUE_EMBED_LEN_MAX]))
-
-#define RB_RVARGC_NEWOBJ_OF(var, T, c, f, s) \
-  T *(var) = (T *)(((f) & FL_WB_PROTECTED) ? \
-                   rb_wb_protected_newobj_of((c), (f) & ~FL_WB_PROTECTED, s) : \
-                   rb_wb_unprotected_newobj_of((c), (f), s))
-
-#define RB_RVARGC_EC_NEWOBJ_OF(ec, var, T, c, f, s) \
+#define RVARGC_NEWOBJ_OF(var, T, c, f, s, ec) \
```

Comment:
Nit: if we're getting rid of `NEWOBJ_OF`, can we rename `RVARGC_NEWOBJ_OF` to just `NEWOBJ_OF`?